### ISSUE TITLE
bug: docker-compose postgres service throws "Operation not permitted"

### ISSUE THREAD

**Issue by myesn** [logto-io/logto#1933]
### Describe the bug

`postgres` service cannot work.

### Expected behavior

Normal work.

### How to reproduce?

docker-compose.yml:
```yaml
version: "3.9"

services:
  app:
    depends_on:
      postgres:
        condition: service_healthy
    image: ghcr.io/logto-io/logto:prerelease
    ports:
      - "443:443"
    environment: # https://docs.logto.io/zh-cn/docs/references/core/configuration/#%E9%80%9A%E7%94%A8
      - ALL_YES=1
      - NO_INQUIRY=0
      - TRUST_PROXY_HEADER=1
      - DB_URL_DEFAULT=postgres://postgres:P@ssw0rd@postgres:5432 # https://www.postgresql.org/docs/14/libpq-connect.html#id-1.7.3.8.3.6
      - ENDPOINT=https://myesn.io
      - PORT=443
      - HTTPS_CERT_PATH=/https/cert/myesn.io.pem
      - HTTPS_KEY_PATH=/https/cert/myesn.io-key.pem
    volumes:
      - ./logto/https/cert:/https/cert

  postgres: # https://github.com/docker-library/docs/blob/master/postgres/README.md#environment-variables
    image: postgres:14.5
    user: postgres
    environment:
      - POSTGRES_USER=postgres
      - POSTGRES_PASSWORD=P@ssw0rd
    ports:
      - "5432:5432"
    healthcheck:
      test: ["CMD-SHELL", "pg_isready"]
      interval: 10s
      timeout: 5s
      retries: 2
    volumes:
      - ./logto/postgres-data:/var/lib/postgresql/data
```

then execute `docker-compose up -d`, the results are as follows:
![image](https://user-images.githubusercontent.com/18598579/190237926-8c3dc8d1-07f3-4111-839d-cb6f75131b4f.png)

When I remove the `user: postgres` property of the `postgres` service, the `postgres` service starts normally, but every few seconds it throws a `FATAL: role "root" does not exist` error，as follows:
![image](https://user-images.githubusercontent.com/18598579/190240661-d769a2ba-9256-4fa4-9ce2-1ce1c935c6c0.png)

### Context

OS: Linux ubuntu 5.4.0-113-generic x86_64

Environment:  

Docker version 20.10.14, build a224086

Docker Compose version v2.2.2

Logto version: [1.0.0-beta.9](https://github.com/orgs/logto-io/packages/container/logto/40782298?tag=1.0.0-beta.9)

Node version: Logto Docker Image

Postgres version: 14.5

Command of starting Logto:

**Comment by gao-sun** [comment]
FYI https://stackoverflow.com/questions/60193781/postgres-with-docker-compose-gives-fatal-role-root-does-not-exist-error

**Comment by gao-sun** [comment]
i think it's not related to Logto. but i'm curious about it too, please let me know if you figure out the fix. :-)

**Comment by myesn** [comment]
Thank you for your reply, I am very sorry, I myself know that this issue is not related to logto. It's just that I spent two hours yesterday in a desperate situation before I posted this issue, so I'm sorry for the trouble I caused you.

The solution is as follows:

The configuration is the same as above, first manually create `postgres-data` directory on the host, then execute `sudo chown -R 999:999 postgres-data` and finally `docker-compose up -d` and you're done.

See [https://github.com/docker-library/postgres/issues/116](https://github.com/docker-library/postgres/issues/116)
